lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2808,6 +2808,14 @@ void generateIntelFPGAAnnotation(const SPIRVEntry *E,
       Out << ":" << Str;
     Out << '}';
   }
+  if (E->hasDecorate(DecorationBankBitsINTEL)) {
+    Out << "{bank_bits:";
+    auto Literals = E->getDecorationLiterals(DecorationBankBitsINTEL);
+    for (size_t I = 0; I < Literals.size() - 1; ++I) {
```

Comment:
```suggestion
    for (auto &Decoration L : Literals) {
```

Reply:
It would make sense if we need to iterate through all the literals. The problem is in a comma symbol, that should be printed after all literals except the last one. That's why there is no range loop here (maybe there is a better solution?)

Reply:
Ah, of course. My bad.